website/docs/docs/cloud/git/setup-azure.md

```diff
@@ -55,12 +55,12 @@ Provide your new app access to Azure DevOps:
 
```

Comment:
above this, i think there's an image that references AD - should that change to ms entra id?

![Screenshot 2024-03-21 at 12 44 56](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/3994901a-1f09-4097-9b1d-90720bc40f1c)

Reply:
got it -- nice catch!